src/main/java/com/github/onsdigital/babbage/template/handlebars/helpers/ArrayHelpers.java

```diff
@@ -18,7 +16,7 @@ public enum ArrayHelpers implements BabbageHandlebarsHelper<Iterable> {
 
     contains {
         @Override
-        public CharSequence apply(Iterable context, Options options) throws IOException {
+        public Object apply(Iterable context, Options options) throws IOException {
```

Comment:
Just wondering if this can use generics rather than simply using Object in order to help with Type safety?

Reply:
Unless this is a requirment of the handlebars library?

Reply:
Discussed with author.